bsdate.go

```diff
@@ -266,6 +282,9 @@ func (d date) isValid() bool {
 	if d.Day <= 0 || d.Day > 32 || d.Month <= 0 || d.Month > 12 || d.Year <= 0 {
 		return false
 	}
+	if reflect.TypeOf(d.Day) == reflect.TypeOf("string") || reflect.TypeOf(d.Month) == reflect.TypeOf("string") || reflect.TypeOf(d.Year) == reflect.TypeOf("string") {
```

Comment:
long line
this change is not mentioned in the description

Reply:
fixed